lib/validator.js

```diff
@@ -62,11 +44,23 @@ module.exports = class Validator {
      * @returns {ValidationResult}
      */
     validateCommitMessage (commitMessage) {
-        if (this._isSingleLineCommitMessage(commitMessage)) {
-            return this._validateSingleLineCommitMessage(commitMessage);
-        } else {
-            return this._validateMultiLineCommitMessage(commitMessage);
+        // If this is a merge commit, just consider it valid.
+        // Otherwise, run the commit message through all configured rules.
+        if (CommitMessageParser.isMergeCommit(commitMessage)) {
```

Comment:
Do you mean passing in the `CommitMessageParser` itself? i.e. dependency inversion

If so, you're probably right. As they're all static methods of the class, it felt a bit odd passing in the class rather than an instance, but I don't have any better reasoning than that.

It would make it easier to write unit tests for ignoring merge commits, and it does feel cleaner... Just feels wrong to pass in a class constructor to a constructor.

You know what I'm just going to do it.